.github/scripts/sync-deps.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Bash Script to sync explicit dependencies from pixi.lock into requirements.txt
+set -euo pipefail
+
+echo "# please sync the dependencies with pyproject.toml and pixi.toml" > requirements.txt
+pixi list --explicit --json | jq -r '.[] | select(.name != "python") | "\(.name)==\(.version)"' >> requirements.txt
```

Comment:
```suggestion
pixi list -e default --explicit --json | jq -r '.[] | select(.name != "python") | "\(.name)==\(.version)"' >> requirements.txt
```

in case you are in a `pixi shell`